scio-macros/src/main/scala/com/spotify/scio/coders/AvroCoderMacros.scala

```diff
@@ -37,7 +37,10 @@ private[coders] object AvroCoderMacros {
     _root_.com.spotify.scio.coders.Coder.beam(
       _root_.org.apache.beam.sdk.coders.AvroCoder.of[$companioned](
         classOf[$companioned],
-        ${companionType}.getClassSchema()
+        classOf[$companioned]
```

Comment:
`getSchema` is a public member, so no need to use reflection here - `new $companioned().getSchema()` should work.

Reply:
Yes indeed